src/praisonai-agents/praisonaiagents/mcp/mcp_http_streaming.py

```
@@ -0,0 +1,207 @@
+    
+    def __init__(self, tool_def: Dict[str, Any], call_func):
+        self.name = tool_def["name"]
+        self.description = tool_def.get("description", "")
+        self.inputSchema = tool_def.get("inputSchema", {})
+        self._call_func = call_func
+        
+    def __call__(self, **kwargs):
+        """Synchronous wrapper for calling the tool."""
+        try:
+            # Check if there's already a running loop
+            loop = asyncio.get_running_loop()
```

Ruff (0.11.9)

Local variable `loop` is assigned to but never used

Remove assignment to unused variable `loop`

(F841)